cmd/bosun/conf/conf.go

```diff
@@ -65,6 +67,23 @@ type Conf struct {
 	squelch         []string
 }
 
+type AuthorizedIPs []*net.IPNet
+
+func (authorizedIPs AuthorizedIPs) Authorized(ip net.IP) (authorized bool) {
+	if len(authorizedIPs) == 0 {
+		return true
+	}
+	if ip.IsLoopback() {
+		return true
+	}
+	for _, ipnet := range authorizedIPs {
```

Comment:
do you know if the net package caches results for Contains? I would hate for this check to become a bottleneck in receiving data.